### PR TITLE
fix: Default grid-template-column value has unintended effect on uncentered justification selection

### PR DESCRIPTION
**Pull request by qweliant**
Resolves unintended side effect mentioned in https://knowledgefutures.freshdesk.com/a/tickets/3665

This fix sets the grid template column repeat css property to auto-fill unless the justification is centered. 

Test plan:

on duqduq change justification labels for content blocks. inspect page to check that the grid-template-columns setting is only auto-fit when trying to center the content.

Unintentional side effect of defaulting to auto-fit
image

Intended effect for non-centered grid items
Screen Shot 2021-06-24 at 12 08 30 PM

### CODE REVIEW

File: client/components/Layout/LayoutPagesCollections.tsx [hunk 1]
```diff
@@ -37,6 +37,7 @@ const resolveItemsFromContent = (
 
 const LayoutPagesCollections = (props: Props) => {
 	const { content, collections, pages } = props;
+	const gridTemplateColumnsCSSFill = content.justify === 'center' ? 'auto-fit' : 'auto-fill';
```
Comment (idreyn) [inline client/components/Layout/LayoutPagesCollections.tsx:40]: Initialisms in camel-cased variable names in our codebase tend not to have all letters capitalized, so we'd more likely write `gridTemplateColumnsCssFill`.

File: client/components/LayoutEditor/layoutEditor.scss [hunk 2]
```diff
@@ -14,7 +14,7 @@
 	.component-wrapper {
 		margin-bottom: 2em;
 		border: 1px solid #ccc;
-	}
+	}
```
Comment (idreyn) [inline client/components/LayoutEditor/layoutEditor.scss:17]: Might as well revert this (I'm working on getting a Prettier-style linter set up for scss so we won't have to worry about this manually)

**Review by idreyn — APPROVED** [review]
LGTM! Some non-critical stuff to look at in the comments. Thanks for the fix.

File: client/components/LayoutEditor/layoutEditor.scss (hunk 2 above)
Comment (qweliant) [reply client/components/LayoutEditor/layoutEditor.scss:17]: not the spacing *shakes fist in air* lol. gotcha

File: client/components/Layout/LayoutPagesCollections.tsx (hunk 1 above)
Comment (qweliant) [reply client/components/Layout/LayoutPagesCollections.tsx:40]: changed it

**Pull request merged**